### PR TITLE
Update cf-231A.mdx

### PR DESCRIPTION
**Pull request by sathu-beep**

### CODE REVIEW

**Comment by sathu-beep** [comment]
@maggie-j-liu 
i implemented your suggestions on the file but it seemed the file was part of usaco guided repo for some reason so it made a new pr

File: cf-231A.mdx [hunk 1]
```diff
@@ -5,7 +5,7 @@ title: Team
 author: Sathvik Chundru
```
Comment (maggie-j-liu) [inline cf-231A.mdx:5]: this file should be inside the solutions folder

Comment (sathu-beep) [reply cf-231A.mdx]: done

Comment (sathu-beep) [reply cf-231A.mdx]: i also moved it into expected knowledge

File: content/1_General/Expected.problems.json [hunk 2]
```diff
@@ -13,6 +13,19 @@
...
+      "uniqueId": "cf-231A",
+      "name": "Team",
+      "url": "https://codeforces.com/problemset/problem/231/A",
+      "source": "CF",
+      "difficulty": "Very Easy",
+      "isStarred": false,
+      "tags": ["Brute Force", "Greedy"],
+      "solutionMetadata": {
+        "kind": "autogen-label-from-site",
+        "site": "CF"
```
Comment (maggie-j-liu) [inline content/1_General/Expected.problems.json:26]: change kind to internal
```suggestion
        "kind": "internal"
```

File: solutions/cf-231A.mdx [hunk 3]
```diff
@@ -0,0 +1,77 @@
+---
+id: cf-231A
+source: CF
+title: Team
+author: Sathvik Chundru
+---
+
+[Official Analysis (C++)](https://codeforces.com/blog/entry/5486)
+
+## Solution (Greedy)
```
Comment (maggie-j-liu) [inline solutions/cf-231A.mdx:10]: i don't think this is greedy...

File: solutions/cf-231A.mdx [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+1 1 1
+1 0 0
+```
+
+The simplest solution is to input three integers per row and check 
+if their sum is greater than or equal to two. 
+
+## Implementation
+
+<LanguageSection>
```
Comment (maggie-j-liu) [inline solutions/cf-231A.mdx:26]: can you add sans's py sol from his comment on the other pr?

File: content/1_General/Expected.problems.json [hunk 5]
```diff
@@ -13,6 +13,19 @@
...
       }
     },
+    {
+      "uniqueId": "cf-231A",
+      "name": "Team",
+      "url": "https://codeforces.com/problemset/problem/231/A",
+      "source": "CF",
+      "difficulty": "Very Easy",
+      "isStarred": false,
+      "tags": ["Brute Force", "Greedy"],
```
Comment (maggie-j-liu) [inline content/1_General/Expected.problems.json:23]: i don't think this is greedy

**Review by maggie-j-liu — CHANGES_REQUESTED** [review]
this pr is trying to merge your branch into the cpinitiative:sathu2 branch, but we need to merge to cpinitiative:master; is there any way you can change that? if not, open a new pr

File: solutions/cf-231A.mdx (hunk 4 above)
Comment (SansPapyrus683) [reply solutions/cf-231A.mdx:26]: lmao that was a one-liner and pretty hard to read here's a more readable one
```py
sols = 0
for _ in range(int(input())):
	sols += sum(int(i) for i in input().spllit()) >= 2
print(sols)
```
not tested but it should work

Comment (sathu-beep) [reply solutions/cf-231A.mdx:26]: um... ok ill add it but it looks very sus to read...

File: content/1_General/Expected.problems.json (hunk 5 above)
Comment (sathu-beep) [reply content/1_General/Expected.problems.json:23]: removed

File: solutions/cf-231A.mdx (hunk 4 above)
Comment (SansPapyrus683) [reply solutions/cf-231A.mdx:26]: better than that one-liner i posted earlier

Comment (maggie-j-liu) [reply solutions/cf-231A.mdx:26]: @sathu-beep can you test it to make sure it works?

Comment (sathu-beep) [reply solutions/cf-231A.mdx:26]: it works

Comment (sathu-beep) [reply solutions/cf-231A.mdx:26]: i added it now @maggie-j-liu 
merge if no more revisions needed

**Comment by sathu-beep** [comment]
@maggie-j-liu i changed it so that its merging into cpinitiative:master and not whatever branch I had

File: solutions/cf-231A.mdx [hunk 6]
```diff
@@ -74,4 +74,15 @@ import java.io.*;
```
Comment (maggie-j-liu) [inline solutions/cf-231A.mdx:74]: can you check that the indentation in the java solution is tabs? indentation looks off on the website
image

File: solutions/cf-231A.mdx [hunk 7]
```diff
@@ -7,7 +7,7 @@ author: Sathvik Chundru
```
Comment (maggie-j-liu) [inline solutions/cf-231A.mdx:7]: add kevin to authors?

File: solutions/cf-231A.mdx [hunk 8]
```diff
@@ -7,7 +7,7 @@ author: Sathvik Chundru
 
 [Official Analysis (C++)](https://codeforces.com/blog/entry/5486)
 
-## Solution (Greedy)
+## Solution
 
 Let's consider the following test case:
```
Comment (maggie-j-liu) [inline solutions/cf-231A.mdx:13]: i don't know if we need this (the test case), what do other people think?

File: solutions/cf-231A.mdx [hunk 9]
```diff
@@ -74,4 +74,15 @@ import java.io.*;
 
 </JavaSection>
 
+<PythonSection>
```
Comment (maggie-j-liu) [inline solutions/cf-231A.mdx]: should be `<PySection>` and `</PySection>` below

**Review by maggie-j-liu — CHANGES_REQUESTED** [review]

File: solutions/cf-231A.mdx (hunk 7 above)
Comment (sathu-beep) [reply solutions/cf-231A.mdx:7]: what is his full name

File: solutions/cf-231A.mdx (hunk 8 above)
Comment (nwatx) [reply solutions/cf-231A.mdx:13]: i think remove it

File: solutions/cf-231A.mdx (hunk 7 above)
Comment (maggie-j-liu) [reply solutions/cf-231A.mdx:7]: Kevin Sheng

File: solutions/cf-231A.mdx (hunk 6 above)
Comment (sathu-beep) [reply solutions/cf-231A.mdx:65]: fixed

File: solutions/cf-231A.mdx (hunk 7 above)
Comment (sathu-beep) [reply solutions/cf-231A.mdx:7]: added him

File: solutions/cf-231A.mdx (hunk 8 above)
Comment (sathu-beep) [reply solutions/cf-231A.mdx]: removed

File: solutions/cf-231A.mdx [hunk 10]
```diff
@@ -0,0 +1,79 @@
+---
+id: cf-231A
+source: CF
+title: Team
+author: Sathvik Chundru, Kevin Sheng
+---
+
+[Official Analysis (C++)](https://codeforces.com/blog/entry/5486)
+
+## Solution
```
Comment (maggie-j-liu) [inline solutions/cf-231A.mdx:10]:
```suggestion
## Explanation
```
to be consistent?

Comment (sathu-beep) [reply solutions/cf-231A.mdx]: done

**Review by maggie-j-liu — APPROVED** [review]

**Comment by nwatx** [comment]
wait, why is this tagged as brute force?

**Comment by maggie-j-liu** [comment]
> wait, why is this tagged as brute force?

Yeah, that can probably be removed too

**Comment by sathu-beep** [comment]
> > wait, why is this tagged as brute force?
> 
> Yeah, that can probably be removed too

removed it as well

**Review by nwatx — APPROVED** [review]

**Comment by sathu-beep** [comment]
@maggie-j-liu @HarryWangATX @dongliu0426 
nathan told me to tell one of the content managers to merge

File: solutions/cf-231A.mdx [hunk 11]
```diff
@@ -0,0 +1,79 @@
...
+
+[Official Analysis (C++)](https://codeforces.com/blog/entry/5486)
+
+## Explanation
+
+The simplest solution is to input three integers per row and check 
+if their sum is greater than or equal to two. 
+
+## Implementation
+
```
Comment (dongliu0426) [inline solutions/cf-231A.mdx:16]: add time complexity

**Review by dongliu0426 — CHANGES_REQUESTED** [review]

File: solutions/cf-231A.mdx (hunk 11 above)
Comment (sathu-beep) [reply solutions/cf-231A.mdx:16]: bro i suck at calculating time and idk latex can u calculate and add it pls

**Comment by sathu-beep** [comment]
thanks neowo

**Comment by sathu-beep** [comment]
@HarryWangATX 
merge 🥺

**Pull request merged**